Plots/Trajectory/NCL_polyg_14.py

```diff
@@ -0,0 +1,96 @@
+"""
+NCL_polyg_14.py
+================
+This script illustrates the following concepts:
+    - Drawing polylines and markers using great circle paths
+    - Using gc_latlon to calculate a great circle path
+    - Attaching polylines and markers to a map plot
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polyg_14.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polyg_14_2_lg.png
+                         https://www.ncl.ucar.edu/Applications/Images/polyg_14_1_lg.png
+"""
+
+
+###############################################################################
+# Import packages:
+import numpy as np
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+from geographiclib.geodesic import Geodesic
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+#Plot
+
+def Plot(color, ext, xext, yext, title, style, pt):
+
+    plt.figure(figsize=(8,8))
+    ax = plt.axes( projection=ccrs.PlateCarree())
+    
+    ax.set_extent(ext, ccrs.PlateCarree())
+    ax.add_feature(cfeature.LAND, color='lightgrey')
+    
+    plt.plot(xext, yext, style, color= color,  transform=ccrs.Geodetic())
+    
+    # start location
+    lon_fr = xext[0]
+    lat_fr = yext[0]
+    # end location
+    lon_to = xext[1]
+    lat_to = yext[1]
+    
+    # This gets geodesic between the two points
+    # WGS84 ellipsoid is used
+    gl = Geodesic.WGS84.InverseLine(lat_fr, lon_fr, lat_to, lon_to) 
+    num_points = 10
```

Comment:
Could this value be added to the argument list for the function instead of hard coded?

Reply:
Updated in most recent push

Reply:
The most recent push still has this hard coded. Could you add it as an argument to the plot function? It would be easier for users to then copy your function and use it this way. If you don't think it's necessary to have it as an argument, you could move this assignment to the beginning of the function with a comment about what it represents.

Reply:
Sorry I should have specified. I meant `num_points`